pkg/issuer/acme/dns/azuredns/azuredns.go

```diff
@@ -104,7 +107,7 @@ func (c *DNSProvider) Present(domain, fqdn, value string) error {
 func (c *DNSProvider) CleanUp(domain, fqdn, value string) error {
 	z, err := c.getHostedZoneName(fqdn)
 	if err != nil {
-		klog.Infof("Error getting hosted zone name for: %s, %v", fqdn, err)
+		c.log.V(logf.WarnLevel).Info("Error getting hosted zone name for:", fqdn, err)
```

Comment:
Maybe log this using `.Error(err, "Error getting hosted zone name", "fqdn", fqdn)`